package-system/zlib/build_zlib_linux.sh

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+# 
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+cmake -S temp/src -B temp/build -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_STANDARD=17 -DCMAKE_C_FLAGS=-fPIC -DSKIP_INSTALL_FILES=YES
```

Comment:
Should add a status code check after the generation as well

```
if [ $? -ne 0 ]
then
    echo "Error generating build"
    exit $?
fi
```

Reply:
ok. I can add this without actually having to rebuild the package, thankfully.

Reply:
Done, although, I've changed it to exit 1 instead of $? becuase by the time it would call exit, the ? is no longer valid.

Reply:
ah thats right, echo would reset it